sdv/metadata/single_table.py

```diff
@@ -666,6 +666,14 @@ def _validate_key(self, column_name, key_type):
                 raise InvalidMetadataError(f"'{key_type}_key' must be a string.")
 
             keys = {column_name} if isinstance(column_name, str) else set(column_name)
+            setting_sequence_as_primary = key_type == 'primary' and column_name == self.sequence_key
+            setting_primary_as_sequence = key_type == 'sequence' and column_name == self.primary_key
+            if setting_sequence_as_primary or setting_primary_as_sequence:
+                raise InvalidMetadataError(
+                    f'The column ({column_name}) cannot be set as {key_type}_key as it is already set as the '
```

Comment:
This line is 111 characters. Not sure why the linter isn't complaining

Reply:
@gsheni It seems to be because we are ignoring the [line-to-long](https://docs.astral.sh/ruff/rules/line-too-long/) rule here
https://github.com/sdv-dev/SDV/blob/f55e51e5b886d6f1f6b989e2ddc730bc835f4e44/pyproject.toml#L206

Reply:
Added that rule back in and fixed the line length